readme.md

```diff
@@ -19,25 +18,27 @@ This project will assess the following learning objectives, in addition to many
 Follow the instructions below to get this project up and running on your own machine:
 
 - Fork and clone this repository.
-- Run `npm install`.
+- Run `npm install` to install the dependencies needed for this project.
 
 To run the tests, you can run the following command:
 
 ```bash
 npm test
 ```
 
-To watch how the code you write affects the application website, you can run the following command. This command will start a server and take over your terminal window. To stop the server from running, you can press `Ctrl + C`.
+To watch how the code you write affects the application website, you can run the following command, which will start a server and take over your terminal window:
 
 ```bash
 npm start
 ```
 
+To stop the server and regain control of your terminal, you can press `Ctrl + C`.
+
 ## Instructions
 
 You are tasked with building functions for an application that will either encode or decode a string using a variety of ciphers. For each cipher, you should make a series of tests using Mocha & Chai to confirm that your cipher works.
 
-All of the functions can be found inside of the `src/` directory. Each function and cipher is described below.
+All of the functions can be found inside of the `src/` directory, and the corresponding test files can be found in `tests/`. Each function and cipher is described below.
```

Comment:
I went ahead and created a `tests/` folder and empty test files for them to write their tests in.